Sources/SwiftFormatPrettyPrint/TokenStreamCreator.swift

```diff
@@ -853,6 +853,7 @@ private final class TokenStreamCreator: SyntaxVisitor {
   }
 
   func visit(_ node: FunctionParameterSyntax) -> SyntaxVisitorContinueKind {
+    after(node.attributes?.lastToken, tokens: .break)
     before(node.firstToken, tokens: .open)
```

Comment:
I'd also recommend leaving this as the first line of the function, so that the relationship between the `.open` group before the first token and the rest of the content is clear to the reader, since this `.open` group should encompass everything else in the parameter node, including the attributes. (`arrangeAttributeList` also calls `before()` on the same node, so ordering here theoretically _does_ matter, even though _practically_ it doesn't in this specific case because the only thing that method inserts is another `.open`.)

Reply:
Sounds good, it did feel a bit odd adding this before the `.open` token, though I wasn't positive I fully understood how these tokens would come together. Will add this right after the `.open` token per your suggestion. Thanks!